Networking/Networking/Mapper/ListMapper.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+
+/// ListMapper: Maps generic WooCommerce REST API Lists
+///
+struct ListMapper<Output: Decodable>: Mapper {
```

Comment:
👏 no more copying mapper for new use case! We can create an issue to replace existing mappers and let the mobile team know.